doc/lsst.scarlet.lite/getting_started.rst

```diff
@@ -0,0 +1,232 @@
+
+>>> # Create a PSF for the final model. This should be a narrow but Nyquist sampled Gaussian
+>>> model_psf = scl.utils.integrated_circular_gaussian(sigma=0.8)
+>>> # Initialized the observation.
+>>> observation = scl.Observation(
+...     data["images"],  # the observed image in each band
+...     data["variance"],  # the variance in each band
+...     1/data["variance"],  # weights to use for calculating the log likelihood
+...     data["psfs"],  # an image of the point spread function in each band
+...     model_psf[None],  # the PSF of the model space
+...     bands=data["filters"],  # the name of each band
+... )
+>>> # Remove the
+>>> del data
+
+.. warning::
+
+    A key difference between scarlet lite and the original `scarlet <https://github.com/lsst/scarlet>`_ module is that scarlet lite assumes that all of the images have been aligned to the same WCS and pixel grid.
+    If this is not the case for your observations, you will either need to reproject them to the same pixel grid or use the full `scarlet <https://github.com/lsst/scarlet>`_ package.
+
+Once an observation has been created a convenience function can be used to display the image, label the sources, and display the PSF:
+
+>>> scl.display.show_observation(observation, norm, centers=centers, psf_scaling="same") # doctest: +ELLIPSIS
+<...>
+
+The ``psf_scaling`` argument determines how the PSF will be displayed. If ``psf_scaling == None`` then the PSF is not displayed at all and if ``psf_scalling == "same"`` then the PSF image will use the same pixel scale as the observation.
+However, sometimes the image might be very large, so setting ``psf_scaling = "native"`` will display the PSF image in it's native pixel scale.
+
+.. _lsst.scarlet.lite-init-sources:
+
+Initializing Sources and Components
+===================================
+
+Like scarlet main, in scarlet lite we make the assumption that all sources can be modeled as a set of components, for example a bulge and a disk to represent a distant galaxy or additional collections of star forming regions in a resolved spiral.
+In general these components can have independent models with their own constraints but typically, in the LSST implementation of scarlet lite, all components are factorized into a 1D spectral array that descibes the intensity in each band, and a 2D morphology array that describes the normalized shape of the component (see :ref:`lsst.scarlet.lite-components`).
+
+.. _lsst.scarlet.lite-sources:
+
+Sources
+-------
+
+Unlike scarlet main, in scarlet lite a :py:class:`~lsst.scarlet.lite.source.Source` is simply a collection of additive components that is responsible for building the joint model of the components, and convenience tools to describe the properties of the joint model like it's bounding box, spectral information, etc.
```

Comment:
Yes, I tried to be consistent. I'll try to find places where I wasn't. As you mentioned in a previous comment, in n(<3?) years references to scarlet main will probably be unnecessary, but for now I think it will be helpful to people migrating to scarlet lite.